exercises/concept/bread-and-potions/.docs/instructions.md

```diff
@@ -21,7 +21,7 @@ RPG.Edible.eat(%RPG.LoafOfBread{}, %RPG.Character{health: 31})
 
 ## 3. Make mana potions edible
 
-Implement the `RPG.Edible` protocol for the `RPG.ManaPotion` item. When eaten, a mana potion gives the character as many mana points as its strength, and produces an empty bottle.
+Implement the `RPG.Edible` protocol for the `RPG.ManaPotion` item. When eaten, a mana potion gives the character as many mana points as the mana's strength, and produces an empty bottle.
```

Comment:
I think it should say the potion’s strength, because mana has no strength.

Reply:
makes sense ... updating